clients/js/test/extensions/permanentDelegate/initializePermanentDelegate.test.ts

```diff
@@ -0,0 +1,41 @@
+import { Account, address, some } from '@solana/web3.js';
+import test from 'ava';
+import { Mint, extension, fetchMint } from '../../../src';
+import {
+  createDefaultSolanaClient,
+  createMint,
+  generateKeyPairSignerWithSol,
+} from '../../_setup';
+
+test('it initializes a mint with permanent delegate', async (t) => {
+  // Given some signer accounts
+  const client = createDefaultSolanaClient();
+  const [authority] = await Promise.all([generateKeyPairSignerWithSol(client)]);
+
+  // And a permanent delegate extension
+  const permanentDelegate = address(
+    '6sPR6MzvjMMP5LSZzEtTe4ZBVX9rhBmtM1dmfFtkNTbW'
+  );
+  const permanentDelegateExtension = extension('PermanentDelegate', {
+    delegate: permanentDelegate,
+  });
+
+  // When we create a mint with this extension
+  const mintAddress = await createMint({
+    authority,
+    client,
+    extensions: [permanentDelegateExtension],
+    payer: authority,
+  });
```

Comment:
Because here, we're testing the initialisation, would you mind using the longer version instead of the `createMint` helper?

Reply:
Sure will update this.

Reply:
@lorisleiva I have updated code with a longer version of mint and rebased it with the main branch. Please review.